code/datums/gamemode/role/ninja.dm

```diff
@@ -83,7 +83,11 @@
 			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Crazed Weeaboo.<br>The crew has insulted glorious Space Nippon. Equipped with your authentic Space Kimono, your Space Katana that was folded over a million times, and your honobru bushido code, you must implore them to reconsider!</span>")
 			to_chat(antag.current, "<span class='danger'>Remember that guns are not honoraburu, and that your katana has an ancient power imbued within it. Take a closer look at it if you've forgotten how it works.</span>")
 		else
-			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Space Ninja.<br>The Spider Clan has been insulted for the last time. Send Nanotrasen a message. You are forbidden by your code to use guns, do not forget!</span>")
+			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Space Ninja! <br>The Spider Clan has been insulted for the last time.</span>")
+			to_chat(antag.current, "Your energy katana cannot be dropped while active, does not conduct electricity, can force open doors, and can teleport behind someone on attack once a minute by using the action button.")
+			to_chat(antag.current, "Your energy glove can drain power from most things that use cells by using an empty hand on them. Some examples are on the right.")
```

Comment:
i can't think of something cell-powered that doesn't have a get_cell()

Reply:
just covering my ass for the oldcoders of the future